src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Subscriptions.razor

```diff
@@ -145,6 +149,14 @@
     GridSort<Subscription> SortBy(Expression<Func<Subscription, string>> sorter)
         => GridSort<Subscription>.ByAscending(sorter);
 
+    protected override void OnInitialized()
+    {
+        if (!string.IsNullOrEmpty(searchFilterQuery))
+        {
+            SearchFilter = searchFilterQuery;
+        }
+    }
+
     protected override async Task OnParametersSetAsync()
     {
         AllSubscriptions = null;
```

Comment:
Maybe `OnParametersSetAsync` should be removed and the code inside moved to `OnInitializedAsync()`?